components/sup/src/manager/mod.rs

```diff
@@ -611,18 +611,7 @@ impl Manager {
             }
         };
 
-        if let Err(e) = service.create_svc_path() {
-            outputln!(
-                "Can't create directory {}: {}",
-                service.pkg.svc_path.display(),
-                e
-            );
-            outputln!(
-                "If this service is running as non-root, you'll need to create \
-                 {} and give the current user write access to it",
-                service.pkg.svc_path.display()
-            );
-            outputln!("{} failed to start", &spec.ident);
```

Comment:
Why was all this removed?

Reply:
It was moved to a `create_svc_path` function